src/helpers.ts

```diff
@@ -13,22 +13,22 @@ export function getkubectlDownloadURL(version: string, arch: string): string {
    switch (os.type()) {
       case 'Linux':
          return util.format(
-            'https://storage.googleapis.com/kubernetes-release/release/%s/bin/linux/%s/kubectl',
+            'https://dl.k8s.io/release/v%s/bin/linux/%s/kubectl`,            
```

Comment:
These strings don't look right mixing ` and '. Change the ones at the end to ' then this looks good to me

Reply:
apologies.  I have updated the quotes.